include/vapor/GrownGrid.h

```diff
@@ -0,0 +1,84 @@
+#ifndef GROWNGRID_H
+#define GROWNGRID_H
+
+/* 
+ * This class represents an essentially 2D grid, but grown to appear like 3D.
+ * It does so by containing a 2D grid in the XY plain, and inserting a fixed value
+ * in the Z dimension in a handful of "useful" functions.
+ *
+ * It also implements pure virtual functions of the Grid class in the simplest possible fashion,
+ * but those functions do not perform any calculation, and should not be used.
+ *
+ * Finally, a DataMgr would not return a GrownGrid at any occasion; rather, this 
+ * class is supposed to be created and used locally by its user.
+ */
+
+#include "vapor/Grid.h"
+#include "vapor/DataMgr.h"
+
+namespace VAPoR
+{
+class VDF_API GrownGrid final: public Grid 
+{
+public:
+
+    // Constructor: pass in the 2D grid to be managed, the data manager that will
+    // later be used to destroy the grid, and the default Z value to be used.
+    // Note: the passed in 2D grid should be queried with the "lock" option on,
+    //       and GrownGrid will take ownership of the 2D grid afterwards.
+    //       I.e., GrownGrid will be responsible deleting the underlying 2D grid.
+    GrownGrid( const VAPoR::Grid* gp, VAPoR::DataMgr* mp, float z );
```

Comment:
Since this is a stopgap fix until we change the behavior of the Grid class how about moving the source files to the vapor application directory to prevent its use from propagating? Also, it would be good to document the problem that it is addressing.

Reply:
Fixed in commit b6525ce